### ISSUE TITLE
check-tags.py missed unclosed tags

### ISSUE THREAD

**Issue by ran-huang** [CharLotteiu/pingcap-docs-checks#5]
`check-tags.py` didn't report unclosed tags in https://github.com/pingcap/blog/pull/642:

![image](https://user-images.githubusercontent.com/59654584/130724798-2c22971b-9b29-4fe6-ac79-953ee18adaa5.png)

**Comment by ran-huang** [comment]
@anotherrachel FYI, we can follow up on this issue here. 

@CharLotteiu I'd like to take this issue and fix the problem. 😺

**Comment by ran-huang** [comment]
### TLDR; 🤣 The script is fine. 

The script catches all the unclosed tags on my local computer:
![image](https://user-images.githubusercontent.com/59654584/132472022-4a30147d-25fb-4dda-9c59-5be05a894357.png)

However, it reports no error in GitHub Actions. The log reads as follows:
![image](https://user-images.githubusercontent.com/59654584/132472278-011a7571-fda4-47c2-9ca8-18a8f9abbd42.png)

It turns out [`ci.yml`](https://github.com/pingcap/blog/blob/c15ffcfda9229fec65c46c5ce19e60dfaf8bae01/.github/workflows/ci.yml#L36) has a syntax error, which results in the strange `checked files are` output:

```
echo "checked files are $(git diff-tree --name-only --no-commit-id -r upstream/master...HEAD -- 'zh/' 'en/' '.md' ':(exclude).github/*')";
```

After I removed this line, GitHub Action ran just as expected (see https://github.com/pingcap/blog/pull/663).

So...for this whole time, `check-tags.py` has never worked in blog, docs-tidb-operator, or docs-dm... 🤣 
Anyway, I shall close the issue now and fix the other two docs repos later.
